packages/workflow/scripts/version.js

```diff
@@ -64,6 +65,23 @@ function prompt() {
 
   const { version } = settings.pkg();
   const parsed = semver.parse(version);
+  const versionArg = yargs.argv.version || yargs.argv._[1]
```

Comment:
This supports both version coming from the flag as well as version specified directly in the command (without a flag)